cmd/dm-ctl/main.go

```diff
@@ -48,8 +48,14 @@ func main() {
 
 	// now, we use checker in dmctl while it using some pkg which log some thing when running
 	// to make dmctl output more clear, simply redirect log to file rather output to stdout
-	log.SetLevelByString("info")
-	log.SetOutputByName("dmctl.log")
+	err = log.InitLogger(&log.Config{
+		File:  "dmctl.log",
```

Comment:
is it needed to _Sync_ log to file?

Reply:
I  thought it,  I even think we don't need log here, so I don't add it